### PR TITLE
Flysky FS-A8S article

### PR DESCRIPTION
**Pull request by sfalexrog**
It's about time we've added the article for the FS-A8S. Right now it's a slightly modified version of my personal note, with proper assets added in place of the photos.

Things that should be done before merging:

- [x] Russian article
- [x] English article
- [x] Fully checked assets
- [x] Formatting checks
- [x] Proofreading

### CODE REVIEW

**Comment by okalachev** [comment]
I believe we could merge it as it and then make the fixes after.

**Comment by sfalexrog** [comment]
We still don't have a way to show a blinking LED state, by the way, which is quite unfortunate

**Comment by sfalexrog** [comment]
Okay, now we have some actual blinking images

File: docs/ru/rc_flysky_a8s.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+> **Note** Если в вашем наборе уже есть кабель для подключения приёмника к полётному контроллеру, совместимому с вашим, переходите к [следующему этапу](#rc_bind).
+
+1. Аккуратно извлеките жёлтый провод из коннектора, идущего к приёмнику. Для того, чтобы вытащить провод, приподнимите острым пинцетом замок коннектора:
+
+    <div class="image-group">
+        <img src="../assets/flysky_a8s/01_remove_cable_fs.png" width=300 class="zoom border" alt="a8s wire removal 1">
+        <img src="../assets/flysky_a8s/02_remove_cable_fs.png" width=300 class="zoom border" alt="a8s wire removal 2">
+    </div>
+
+2. (При использовании полётного контроллера Pixracer) Извлеките 2 провода (зелёный и коричневый) из 5-пинового коннектора для полётного контроллера:
```
Comment (okalachev) [inline docs/ru/rc_flysky_a8s.md:18]: Вообще такое использование скобок это немного не по-русски.

http://orthographia.ru/punctuatio.php?sid=128

Comment (sfalexrog) [reply docs/ru/rc_flysky_a8s.md]: И то правда. Попробовал поправить, не знаю, получилось ли лучше.

File: docs/ru/rc_flysky_a8s.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    </div>
+
+2. [При использовании полётного контроллера Pixracer] Извлеките 2 провода - зелёный и коричневый - из 5-пинового коннектора для полётного контроллера:
+
+    <div class="image-group">
+        <img src="../assets/flysky_a8s/03_remove_cable_pixracer.png" width=300 class="zoom border" alt="pixracer wire removal 1">
+        <img src="../assets/flysky_a8s/04_remove_cable_pixracer.png" width=300 class="zoom border" alt="pixracer wire removal 2">
+    </div>
+
+3. [При использовании полётного контроллера COEX Pix] Извлеките зелёный (или синий, в зависимости от комплектации) провод из 4-пинового коннектора для полётного контроллера:
```
Comment (okalachev) [inline docs/ru/rc_flysky_a8s.md:25]: Я бы оформил это место немного по-другому, но потом покажу, как.

**Comment by sfalexrog** [comment]
Okay, merging as is

**Pull request merged**